meetups/Meetup_Best_Practices.md

```diff
@@ -0,0 +1,52 @@
+## Meetups Best Practices
+
+This page allows you to start a new meetup group based on the experience of existing ones. <br>
+Here is a list of suggestions and tips to get you started:
+
+* **Create the meetup**
+
```

Comment:
Maybe i should add a word on the naming of the meetup group ? 
`Cloud Native Computing <Location>` given the current meetups list.
What do you think @idvoretskyi ?

Reply:
@Smana we also accept Meetup groups with the different names, reflecting specific CNCF project (eg. Kubernetes Meetup Group or Prometheus Meetup Group).

Reply:
right, forget it then :)

Reply:
Feel free to add this as a recommendation.

Reply:
i added a recommendation, please tell me if that's ok for you @idvoretskyi